src/xcode/ENA/ENA/Source/Models/ExposureWindowsMetadata/ExposureWindowsMetadataService.swift

```diff
@@ -0,0 +1,73 @@
+////
+// 🦠 Corona-Warn-App
+//
+
+import Foundation
+
+class ExposureWindowsMetadataService {
+	
+	// MARK: - Internal
+
+	func collectExposureWindows(from riskCalculation: RiskCalculationProtocol, store: Store) {
+		guard let calculation = riskCalculation as? RiskCalculation else {
+			Log.debug("Instance of riskCalculation couldn't be casted into RiskCalculation type", log: .ppa)
+			return
+		}
```

Comment:
maybe we should think about extending the protocol so you don't need that typecast
Might be also helpful for testing?

Reply:
Yeah i agree, i will do that

Reply:
After extending the protocol you do not need the typecast / guard anymore.